jp2_online/templates/captura/create_integrante_form.html

```diff
@@ -0,0 +1,4 @@
+<form action="{% url 'captura:create_integrante' familia.id %}" method="post">
```

Comment:
Because by making it a form, we enable the check for CSRF token, this also forces de creation of the integrante to use a POST method, preventing people from typing the string randomly into their browsers, and creating a new integrante.

This is extremely important as some browsers preemptively load a page, and may trigger the functions in it, even if the user didn't actually visit de page.